data/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { createStore } from 'redux';
+import { flowRight } from 'lodash';
+
+/**
+ * Module constants
+ */
+const reducers = [];
+const enhancers = [];
+if ( window.__REDUX_DEVTOOLS_EXTENSION__ ) {
```

Comment:
Do we add checks for window existance in other places?

Reply:
Haven't seen any. Do we expect our modules to work in Node Context?

Reply:
Hmm, I guess not. Old habbit from Calypso days 😎